pages/package/[...packageString]/ResultPage.js

```diff
@@ -3,7 +3,7 @@ import Analytics from '../../../client/analytics'
 
 import ResultLayout from '../../../client/components/ResultLayout'
 import BarGraph from '../../../client/components/BarGraph'
-import AutocompleteInput from '../../../client/components/AutocompleteInput'
+import { AutocompleteInput } from '../../../client/components/AutocompleteInput/AutocompleteInput'
```

Comment:
Let's use a `index.ts` re-export to keep the imports clean. 
`components/AutocompleteInput/AutocompleteInput` reads odd

Reply:
✅